examples/ruby/grpc-demo.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |s|
   s.platform      = Gem::Platform::RUBY
 
   s.add_dependency 'grpc', '~> 1.0'
+  s.add_dependency 'googleauth', '>= 0.5.1', '< 0.7'
```

Comment:
Move googleauth gem dependency to development dependency.  What do you think?

https://github.com/grpc/grpc/pull/15356/commits/fd7c4d5f06219b9979412d2242b3f4c1074dbbff

Reply:
AFAIK moving `googleauth` to be a development dependency won't eliminate the risk that this breaks people that are relying on `googleauth` being in there transitive dependencies (I haven't actually verified this is an issue yet, but I'm just a little skiddish about this).

Reply:
You're right. Well then, How about we bump up a version of grpc-ruby to 2.0.0  and write this break  to Changelog? but I have no idea about the policy of versioning about grpc-ruby. Is it difficult or too caring about?

Reply:
Though its definitely possible that we wouldn't even break anyone by this, maybe it would be useful to file an issue along the lines of "grpc-ruby can drop its googleauth dependency, all in favor?", and then wait for a little bit to see if there's any input.